target/linux/ipq806x/files-5.4/arch/arm/boot/dts/qcom-ipq8064-ea8500.dts

```diff
@@ -279,6 +279,7 @@
 			0x00004 0x7600000   /* PAD0_MODE */
 			0x00008 0x1000000   /* PAD5_MODE */
 			0x0000c 0x80        /* PAD6_MODE */
+			0x00010 0x2613a0    /* PWS_REG */
```

Comment:
Is this whole change specific to kernel 5.4?
If yes, please drop the DTS file for kernel 4.19, as it won't work anymore.
If not, please also apply the changes to the 4.19 file.

Reply:
It isn't kernel version specific. I added line for 4.19.